core/string/print_string.h

```diff
@@ -61,6 +61,11 @@ extern void __print_line_rich(String p_string);
 extern void print_error(String p_string);
 extern void print_verbose(String p_string);
 
+// With no arguments, print_line() outputs a newline.
+inline void print_line() {
+	__print_line(stringify_variants(""));
```

Comment:
`stringify_variants` seems unnecessary:

```suggestion
	__print_line(String());
```

Reply:
Thanks! Yes, `String()` is definitely simpler!

Reply:
Can I do this instead?

```
__print_line("");
```

Other parts of Godot engine pass in "" when an empty String is expected. 

For example in `ustring.cpp`:

```c++
Vector<String> ret;
...
	if (p_allow_empty) {
		ret.push_back("");
	}
```